README.md

```diff
@@ -21,3 +21,22 @@ and send it to your friends :)
 
 note, that the api is not yet complete,
 this is just a, maybe already useful, proof-of-concept for now.
+
+## use type-checking and completion
+
+If you are using vscode you can have autocompletion and type-checking even without using typescript.
+
+just add these two lines to your javascript source files:
+
+```js
+//@ts-check
+/** @typedef {import('./webxdc').WEBxDC} WEBxDC */
```

Comment:
why this mixed case here? in code we were using mainly just `webxdc`, `Webxdc` or `WEBXDC` so far.

Reply:
we checked the PR at the same time hehe, 

I don't have any experience with type script, but it is useful even just reading this `*.d.ts` file I get an idea of the types and expected stuff

Reply:
yip, it is kind of documentation, similar to a header, this is nice.

Reply:
we can rename it to Webxdc everywhere it is mentioned.